From 83138dacd3e0e89fe0180c9d805689f7b699aa51 Mon Sep 17 00:00:00 2001
Message-Id: <patch-v3-26.29-83138dacd3e-20220413T193749Z-avarab@gmail.com>
In-Reply-To: <cover-v3-00.29-00000000000-20220413T193749Z-avarab@gmail.com>
References: <cover-v2-00.25-00000000000-20220325T182534Z-avarab@gmail.com>
	<cover-v3-00.29-00000000000-20220413T193749Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Wed, 13 Apr 2022 14:42:43 +0200
Subject: [PATCH v3 26/29] CI: set SANITIZE=leak in MAKEFLAGS directly
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

As in the preceding commit's change to do likewise for CC, let's not
set SANITIZE in the environment. When I added the "linux-leaks" job in
in 956d2e4639b (tests: add a test mode for SANITIZE=leak, run it in
CI, 2021-09-23) I missed that I could do that.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 ci/lib.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ci/lib.sh b/ci/lib.sh
index c73b107d9c7..b439c1a4f82 100755
--- a/ci/lib.sh
+++ b/ci/lib.sh
@@ -201,7 +201,7 @@ linux-musl)
 	;;
 linux-leaks)
 	CC=gcc
-	setenv --build SANITIZE leak
+	MAKEFLAGS="$MAKEFLAGS SANITIZE=leak"
 	setenv --test GIT_TEST_PASSING_SANITIZE_LEAK true
 	;;
 esac
-- 
2.36.0.rc2.843.g193535c2aa7

